docs/DeveloperGuide.md

```diff
@@ -658,21 +658,21 @@ a desktop app.
+| Priority | As a …​                                                   | I want to …​                                          | So that I can…​                                                             |
+|----------|-----------------------------------------------------------|-------------------------------------------------------|-----------------------------------------------------------------------------|
+| `* * *`  | CS student at NUS                                         | add person contacts with details                      | create a comprehensive network of contacts for academic and career purposes |
+| `* * *`  | CS student at NUS                                         | add company contacts with details                     | track and manage potential job and internship opportunities                 |
+| `* * *`  | CS student at NUS                                         | view all my contacts, including persons and companies | easily access and manage my network and opportunities                       |
+| `* * *`  | CS student at NUS                                         | delete person contacts from my list                   | keep my network current and focused on relevant connections                 |
+| `* * *`  | CS student at NUS                                         | delete company contacts from my list                  | focus on companies and opportunities that are most relevant to me           |
+| `* * *`  | CS student at NUS                                         | edit details of person contacts                       | update information as my network evolves and changes                        |
+| `* * *`  | CS student at NUS                                         | edit details of company contacts                      | keep track of changes in company details and opportunities                  |
+| `* * *`  | CS student at NUS                                         | add internships to company contacts                   | organize and track my internship applications with specific companies       |
+| `* * *`  | CS student at NUS                                         | delete internships from company contacts              | focus on current and relevant internship opportunities                      |
+| `* * *`  | CS student at NUS                                         | view details of specific persons or companies         | access detailed information quickly for networking or application purposes  |
+| `* * *`  | CS student at NUS                                         | find people or companies using search criteria        | locate specific contacts or companies efficiently                           |
+| `* * *`  | CS student at NUS                                         | sort companies based on upcoming internship dates     | prioritize my preparation and application process upcoming internships      |
```

Comment:
Missing an `of` here? (`application process of upcoming internships`)

Reply:
Yep thanks for spotting haha